pkg/execution/datasource/datasource_graphql.go

```diff
@@ -238,13 +238,43 @@ func (g *GraphQLDataSourcePlanner) EnterField(ref int) {
 	}
 }
 
+func (g *GraphQLDataSourcePlanner) EnterArgument(ref int) {
+	variableValue := g.Operation.ArgumentValue(ref)
+	if variableValue.Kind != ast.ValueKindVariable {
+		return
+	}
+
+	variableName := g.Operation.VariableValueNameBytes(variableValue.Ref)
+	definitionRef, exists := g.Operation.VariableDefinitionByNameAndOperation(g.nodes[0].Ref, variableName)
```

Comment:
Is it correct that we always reffer to 0 node?
```
g.nodes[0].Ref
```

Reply:
the function `VariableDefinitionByNameAndOperation(operationDefinition int, name ByteSlice)` requires the ref of the operation definition, which is always the first node in the slice